janus_server/src/aggregator.rs

```diff
@@ -835,10 +937,12 @@ impl VdafOps {
                 .hpke_keys
                 .get(&report_share.encrypted_input_share.config_id())
                 .ok_or_else(|| {
-                    warn!(
+                    error!(
```

Comment:
Since this would result from bad input from a client rather than a mistake on our end, I think it ought to be no higher than `warn!`.